WordPress/Classes/Utility/AB Testing/ABTest.swift

```diff
@@ -2,6 +2,7 @@ import AutomatticTracks
 
 enum ABTest: String, CaseIterable {
     case unknown = "unknown"
+    case landInTheEditorPhase1 = "landInTheEditorPhase1"
```

Comment:
This experiment name is just a suggestion for now, and open to change if desired.

Reply:
The naming looks good to me @mkevins!
@ob1-dev do you think this is a good name for an ExPlat experiment or there are conventions we should follow for this?

Reply:
On further investigating, I think the string of the experiment name should be lowercase snake case. I'll push a change to correct this.

Reply:
Updated with https://github.com/wordpress-mobile/WordPress-iOS/pull/17429/commits/0d03fa60f2030ba0789c1dfd583d99baa65d3504